__tests__/pet.test.js

```diff
@@ -0,0 +1,87 @@
+const Pet = require('../src/pet');
+
+
+describe('constructor', () => {
+    xit('returns an object', () => {
+      expect(new Pet('Fido')).toBeInstanceOf(Object);
+    });
+
+    describe('constructor', () => {
+      
+        xit('sets the name property', () => {
+          const pet = new Pet('Fido');
+      
+          expect(pet.name).toEqual('Fido');
+        });
+      });
```

Comment:
So, there are several things to talk about here:
- The way the describe and test blocks is used (often with the name "constructor") could be organised a little better. The `describe` block is there to provide a wrapper around a group of tests that are similar and make sense to be tested together. In this case you have several tests that are wrapped in a single `describe` block, when they really should all be housed in a single `describe`. EG
```
describe("pet.js", () => {
  describe("constructor", () => {
      it("returns an object", () => {
        // logic here
      });

      it("sets the name property", () => {
        // logic here
      });

      it("has a initial age of 0", () => {
        // logic here
      });
    });
});
```
Here the whole suite is wrapped in a `describe` with the name of the file, and all the values that should be tested to determine the constructor is working are in their own `describe` block. Hopefully this makes sense! When this is done as cleanly as possible the output on the terminal running running the tests should read nice and clearly and it should be obvious what each line is doing
- Most of the tests are marked to be skipped, as the `it` blocks were all prefixed with `x`. Tests should always be checked are running properly before submitting for review
- Empty lines. There are a LOT of empty lines here that dont need to be. Conventionally there should be a blank line between each test and each describe block, but thats about it
- Indentation. There are several tests which are indented way too far, they should be put in just one layer in from their housing `describe` block and no more. Also the level changes throughout the file, sometimes it uses 2 spaces and others 4. Theres no absolute correct answer, but whichever one you use it should be the same throughout the whole project